book/SUMMARY.md

```diff
@@ -8,6 +8,16 @@
   * [Windows WSL2](/docs/setup/WindowsWSL.md)
   * [Ubuntu](/docs/setup/Ubuntu.md)
   * [MacOS including M1](/docs/setup/MacOS.md)
-
+  * [Local Plutus API documentation (Optional)](/docs/setup/buildDocumentation.md)
```

Comment:
CC: @Dauta  and @dino- .

Would that not be a better fit under "Additional Configuration", instead of Getting Started?

Reply:
Yeah, I'd say so as well. Additional or miscellaneous.

Reply:
Sorry, first time contributing on Github. Moved under Additional Configuration